build/uploadperftobenchview.sh

```diff
@@ -63,6 +63,8 @@ else
     GIT_BRANCH_WITHOUT_ORIGIN=$GIT_BRANCH
 fi
 
+timeStamp=`date +%Y-%m-%dT%H:%M:%S%:z`
```

Comment:
It should be:
```bash
timeStamp=`date --utc '+%Y-%m-%dT%H:%M:%SZ'`
```

Reply:
I don't think it matters; both appear correct:
date --utc '+%Y-%m-%dT%H:%M:%SZ'
2018-10-02T22:33:43Z
date +%Y-%m-%dT%H:%M:%S%:z
2018-10-02T15:34:05-07:00